demos/serverless/src/handlers.js

```diff
@@ -208,3 +208,105 @@ exports.end = async(event, context, callback) => {
   }).promise();
   callback(null, response);
 };
+
+exports.logs = async (event, context) => {
+
+  console.log(event);
+  var sequenceToken = "";
+  const logGroupName = "BrowserLogs";
+  var response = {
+    "statusCode": 200,
+    "headers": {},
+    "body": '',
+    "isBase64Encoded": false
+  };
+  try {
+    if (typeof JSON.parse(event.body) === 'undefined' || typeof JSON.parse(event.body).logs === 'undefined') {
+            response.body = "Empty";
+      return response;
+    } else {
+
+      var logs_arr = JSON.parse(event.body).logs;
+      //iterate through this array to get each jsons
+      var logSeqNo;
+      var timestamp;
+      var meetingID;
+      var attendeeID;
+      var logLevelType;
+      var appName;
+      var logMessage;
+
+
+      var cloudwatchlogs = new AWS.CloudWatchLogs({
+        apiVersion: '2014-03-28'
+      });
+      for (let i = 0; i < logs_arr.length; i++) {
+        logSeqNo = JSON.parse(logs_arr[i]).logSeqNo;
+        timestamp = JSON.parse(logs_arr[i]).timestamp;
+        meetingID = JSON.parse(logs_arr[i]).meetingID;
+        attendeeID = JSON.parse(logs_arr[i]).attendeeID;
+        logLevelType = JSON.parse(logs_arr[i]).logLevelType;
+        appName = JSON.parse(logs_arr[i]).appName;
+        logMessage = JSON.parse(logs_arr[i]).logMessage;
+
+        var msg = `${logSeqNo} AttendeeID: ${attendeeID} [${logLevelType}] ${logMessage}`;
+
+        var par = {
+          "logGroupName": logGroupName,
+          "logStreamNamePrefix": meetingID
+        }
+        if (sequenceToken != "") {
+          par = {
+            "logGroupName": logGroupName,
+            "logStreamNamePrefix": meetingID,
+            "nextToken": sequenceToken
+          }
+        }
+
+        var logStreams = await cloudwatchlogs.describeLogStreams(par).promise();
+
+        //if meetingID does not exist as logStream
+        if (logStreams.logStreams.length == 0){
```

Comment:
[**Unlikely scenario but should be handled**]: What would happen if there are log streams with meetinID as prefix but there is not logstream with name as meetingID (not just prefix). You should check for that as well and create a logstream.